aiida/engine/processes/workchains/workchain.py

```diff
@@ -365,7 +368,7 @@ def on_exiting(self) -> None:
             self.logger.exception('exception in _store_nodes called in on_exiting')
 
     @Protect.final
-    def on_wait(self, awaitables: t.Sequence[Awaitable]):
+    def on_wait(self, awaitables: t.Sequence[t.Awaitable]):
```

Comment:
Is this correct? The `Awaitable` is something we define. How does this exist in the `typing` module, and even if it does, that is now what it means does it?

Reply:
Well ha, this is what I mentioned in the team meeting; the thing we refer to as "awaitables" in `aiida.Workchain` have nothing to do with the awaitables in python and plumpy 😒, i.e. things that can be used like `await awaitable`

In plumpy you can see how the Workchain uses them: https://github.com/aiidateam/plumpy/blob/f2dbc1dcc0581c7b6b3438029eeca866df57a24b/src/plumpy/workchains.py#LL57C39-L57C39
None of this is used in aiida